src/modules/UI/components/FlipInput/FlipInput2.ui.js

```diff
@@ -280,18 +292,28 @@ export class FlipInput extends Component<FlipInputOwnProps, State> {
           selectionColor="white"
           returnKeyType="done"
           underlineColorAndroid={'transparent'}
-          ref={ref => {
-            this.textInputFront = ref
-          }}
-          onFocus={() => this.setState({ textInputFrontFocus: true })}
-          onBlur={() => this.setState({ textInputFrontFocus: false })}
+          ref={this.getTextInputFrontRef}
+          onFocus={this.textInputFrontFocusTrue()}
+          onBlur={this.textInputFrontFocusFalse()}
```

Comment:
Should these really have the parentheses at the end? Won't they be called everytime the component is renders, and the return value will be what's executed for `onFocus` or `onBlur`? And neither of those methods have a return value at all (they just re-set state).